tests/test_house_style.py

```
@@ -0,0 +1,83 @@
+        assert "Paarformen" in BASEL_STADT_HOUSE_STYLE
+        assert "Gendersternchen" in BASEL_STADT_HOUSE_STYLE
+        assert "Bürger*innen" in BASEL_STADT_HOUSE_STYLE
+        assert "Bürger·innen" in BASEL_STADT_HOUSE_STYLE
+        assert "Bürger/-innen" in BASEL_STADT_HOUSE_STYLE
+        assert "Kundendienst" in BASEL_STADT_HOUSE_STYLE
+
+    def test_typografie_and_orthografie(self) -> None:
+        assert "immer «ss», nie «ß»" in BASEL_STADT_HOUSE_STYLE
+        assert BASEL_STADT_HOUSE_STYLE.count("ß") == 1
+        assert "« »" in BASEL_STADT_HOUSE_STYLE
+        assert "‹ ›" in BASEL_STADT_HOUSE_STYLE
```

Ruff (0.16.1)

[warning] String contains ambiguous `‹` (SINGLE LEFT-POINTING ANGLE QUOTATION MARK). Did you mean `<` (LESS-THAN SIGN)?

(RUF001)